odoo/http.py

```diff
@@ -538,6 +541,12 @@ def get_response(self, as_attachment=None, **send_file_kwargs):
 
         res = _send_file(self.path, **send_file_kwargs)
 
+        if immutable and res.cache_control:
+            if hasattr(res.cache_control, 'immutable'):  # werkzeug1.0
+                res.cache_control.immutable = True
```

Comment:
ResponseCacheControl is a dict. We can set the directive key manually, it will then work across werkzeug versions, even without the property.
(and the browser will then feel free to ignore it ^^)

Cfr. https://github.com/pallets/werkzeug/issues/1185#issuecomment-532741515

```suggestion
            # cache_control is a dict. None means set for a bool option.
            res.cache_control["immutable"] = None
```